ConfigurationSystem/private/AutoResourceTools/SETypes.py

```diff
@@ -176,7 +185,7 @@ def __new__(cls, se, se_info, srms, xrootd_ports, vo, vo_info, existing_ses=None
         return super(SE, cls).__new__(cls,
                                       DiracName=dirac_name,
                                       AccessProtocols=tuple(aps),
-                                      Host=se if aps else None,
+                                      Host=se if not aps else None,
```

Comment:
@sfayer  Note this is actually nothing to do with the naming fix but rather brings the behavior back to the original script of only adding Host attribute to the SE if no AccessProtocol sections are available. You may have noted that it was otherwise adding the host to the SE section regardless.